src/Symfony/Component/HttpKernel/Tests/KernelTest.php

```diff
@@ -761,6 +761,24 @@ public function testTerminateDelegatesTerminationOnlyForTerminableInterface()
         $kernel->terminate(Request::create('/'), new Response());
     }
 
+    public function testKernelRootDirNameStartingWithANumber()
+    {
+        $kernel = $this->getMockBuilder('Symfony\Component\HttpKernel\Tests\Fixtures\KernelForTest')
+            ->disableOriginalConstructor()
+            ->setMethods(array('getRootDir'))
+            ->getMock();
+
+        $kernel
+            ->expects($this->any())
+            ->method('getRootDir')
+            ->will($this->returnValue(__DIR__.'/Fixtures/123'));
+
+        $kernel->__construct('dev', false);
+
+        $kernel->boot();
+        $this->assertInstanceOf('Symfony\Component\DependencyInjection\ContainerInterface', $kernel->getContainer());
```

Comment:
would be much better to write the test in a way which does not trigger a fatal error on failure, as this forbids running other tests and to get the proper PHPUnit reporting.